.phpunit.result.cache

```diff
@@ -0,0 +1 @@
thentication\\AuthenticationSuccessHandlerTest::testOnAuthenticationSuccess":0.006,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\AuthenticationSuccessHandlerTest::testHandleAuthenticationSuccess":0,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\AuthenticationSuccessHandlerTest::testHandleAuthenticationSuccessWithGivenJWT with data set #0":0.001,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\AuthenticationSuccessHandlerTest::testHandleAuthenticationSuccessWithGivenJWT with data set #1":0.001,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\AuthenticationSuccessHandlerTest::testHandleAuthenticationSuccessWithGivenJWT with data set #2":0,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\AuthenticationSuccessHandlerTest::testRememberDeviceCookieIsSetAfterAuthenticationSuccess":0.001,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\AuthenticationSuccessHandlerTest::testRememberDeviceCookieIsReplacedAfterNewAuthenticationSuccess":0.001,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\Provider\\TwoFactorJWTProviderTest::testSupports with data set \"UsernamePasswordToken not supported\"":0.003,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\Provider\\TwoFactorJWTProviderTest::testSupports with data set \"TwoFactorJWTToken is supported\"":0,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\Provider\\TwoFactorJWTProviderTest::testCheckAuthentication with data set \"Incorrect User gives bad credentials\"":0.001,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\Provider\\TwoFactorJWTProviderTest::testCheckAuthentication with data set \"No password gives bad credentials\"":0,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\Provider\\TwoFactorJWTProviderTest::testCheckAuthentication with data set \"Setup 2FA no secret\"":0,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\Provider\\TwoFactorJWTProviderTest::testCheckAuthentication with data set \"Setup 2FA not confirmed\"":0,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\Provider\\TwoFactorJWTProviderTest::testCheckAuthentication with data set \"Verify if presented but not confirmed\"":0.001,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\Provider\\TwoFactorJWTProviderTest::testCheckAuthentication with data set \"Present challenge\"":0,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\Provider\\TwoFactorJWTProviderTest::testCheckAuthentication with data set \"Incorrect code\"":0.001,"ConnectHolland\\SecureJWTBundle\\Tests\\Security\\Http\\Authentication\\Provider\\TwoFactorJWTProviderTest::testCheckAuthentication with data set \"Correct code\"":0,"ConnectHolland\\SecureJWTBundle\\Tests\\Swagger\\LoginDecoratorTest::testDocumentsAuthentication":0.001}}
```

Comment:
Does this file really need to be committed?

Reply:
Oops, that wasn't supposed to happen 😅